### ISSUE TITLE
[HyperlinkButton] AutomationProperties.HelpText is not announced by Narrator

### ISSUE THREAD

**Issue by niels9001** [microsoft/microsoft-ui-xaml#5428]
**Describe the bug**
A HyperlinkButton has a AutomationProperties.HelpText property that's used by Narrator to announce additional text or instructions to the user. This is a accessibility issue.

**Steps to reproduce the bug**
1. Add a HyperlinkButton
2.  Set the AutomationProperties.Name to "My button"
3. Set the AutomationProperties.HelpText to "It can be clicked"
4. Turn on Narrator and set the focus on the HyperlinkButton.

`    <StackPanel HorizontalAlignment="Center" Orientation="Horizontal" Spacing="12">`
`   <Button Content="Button" AutomationProperties.Name="This is a button" AutomationProperties.HelpText="It can be clicked"/> `
`<SplitButton Content="SplitButton" AutomationProperties.Name="This is a splitbutton" AutomationProperties.HelpText="It can be clicked"/> `
`<HyperlinkButton Content="HyperlinkButton" AutomationProperties.Name="This is a hyperlinkbutton" AutomationProperties.HelpText="It can be clicked"/> `
`</StackPanel>`

**Expected behavior**
Narrator describes the object as: "My button - hyperlinkbutton - it can be clicked"

Actual result: 
Narrator describes the object as: "My button - hyperlinkbutton"

**Version Info**
2.6.1

**Comment by niels9001** [comment]
We have two issues in PowerToys that is blocked by this: https://github.com/microsoft/PowerToys/issues/12028 and https://github.com/microsoft/PowerToys/issues/12023
